File: 2507-number-of-common-factors/number-of-common-factors.js
/**
 * @param {number} a
 * @param {number} b
 * @return {number}
 */
var commonFactors = function(a, b) {
    let small = Math.min(a,b);
    let count = 0
    for(let i = 1; i <= small; i++){
        if(a % i == 0 && b % i == 0){
        count++;
        }
    }
    return count;
};

File: 2507-number-of-common-factors/number-of-common-factors.py
class Solution:
    def commonFactors(self, a: int, b: int) -> int:
        count = 0
        x = min(a,b)
        for i in range(1,x+1):
            if a % i == 0 and b % i == 0:
                count+=1
        return count        
        